Samples/DeploymentManager/cs-winui/SampleApp (Package)/Package.appxmanifest

```diff
@@ -0,0 +1,48 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  IgnorableNamespaces="uap rescap">
+
+  <Identity
+    Name="WindowsAppRuntime.Sample.Cs.DeploymentManager"
+    Publisher="CN=Microsoft Corporation"
+    Version="1.0.0.0" />
+
+  <Properties>
+    <DisplayName>SampleApp (Package)</DisplayName>
+    <PublisherDisplayName>Microsoft Corporation</PublisherDisplayName>
+    <Logo>Images\StoreLogo.png</Logo>
+  </Properties>
+
+  <Dependencies>
+    <TargetDeviceFamily Name="Windows.Universal" MinVersion="10.0.17763.0" MaxVersionTested="10.0.19041.0" />
+    <TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.17763.0" MaxVersionTested="10.0.19041.0" />
+  </Dependencies>
+
+  <Resources>
+    <Resource Language="x-generate"/>
+  </Resources>
+
+  <Applications>
+    <Application Id="App"
+      Executable="$targetnametoken$.exe"
+      EntryPoint="$targetentrypoint$">
+      <uap:VisualElements
+        DisplayName="SampleApp (Package)"
+        Description="SampleApp (Package)"
+        BackgroundColor="transparent"
+        Square150x150Logo="Images\Square150x150Logo.png"
+        Square44x44Logo="Images\Square44x44Logo.png">
+        <uap:DefaultTile Wide310x150Logo="Images\Wide310x150Logo.png" />
+        <uap:SplashScreen Image="Images\SplashScreen.png" />
+      </uap:VisualElements>
+    </Application>
+  </Applications>
+
+  <Capabilities>
+    <rescap:Capability Name="runFullTrust" />
+  </Capabilities>
+</Package>
```

Comment:
if [input](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=571).[Paths](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=410) == nil {
		return nil, [missingExcludeDeps](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=579)
	}

	var [missingDeps](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=1;l=599?gsn=missingDeps&gs=kythe%3A%2F%2Fandroid.googlesource.com%2Fplatform%2Fsuperproject%3Flang%3Dgo%3Fpath%3Dbuild%2Fsoong%2Fandroid%23var%2520%255Bpaths.go%2523599%255D.missingDeps) [][string](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=1;l=599?gsn=string&gs=kythe%3A%2F%2Fgolang.org%3Flang%3Dgo%23string%2523builtin)

	[expandedSrcFiles](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=1;l=601?gsn=expandedSrcFiles&gs=kythe%3A%2F%2Fandroid.googlesource.com%2Fplatform%2Fsuperproject%3Flang%3Dgo%3Fpath%3Dbuild%2Fsoong%2Fandroid%23var%2520%255Bpaths.go%2523601%255D.expandedSrcFiles) := make([Paths](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=340), 0, len([input](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=571).[Paths](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=410)))
	for _, [s](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=1;l=602?gsn=s&gs=kythe%3A%2F%2Fandroid.googlesource.com%2Fplatform%2Fsuperproject%3Flang%3Dgo%3Fpath%3Dbuild%2Fsoong%2Fandroid%23var%2520%255Bpaths.go%2523602%255D.s) := range [input](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=571).[Paths](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=410) {
		[srcFiles](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=1;l=603?gsn=srcFiles&gs=kythe%3A%2F%2Fandroid.googlesource.com%2Fplatform%2Fsuperproject%3Flang%3Dgo%3Fpath%3Dbuild%2Fsoong%2Fandroid%23var%2520%255Bpaths.go%2523603%255D.srcFiles), [err](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=1;l=603?gsn=err&gs=kythe%3A%2F%2Fandroid.googlesource.com%2Fplatform%2Fsuperproject%3Flang%3Dgo%3Fpath%3Dbuild%2Fsoong%2Fandroid%23var%2520%255Bpaths.go%2523603%255D.err) := [expandOneSrcPath](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=638)([sourcePathInput](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=628){
			[context](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=629):          [input](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=571).[Context](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=409),
			[path](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=630):             [s](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=602),
			[expandedExcludes](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=631): [expandedExcludes](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=574),
			[includeDirs](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=632):      [input](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=571).[IncludeDirs](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=412),
		})
		if [depErr](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=1;l=609?gsn=depErr&gs=kythe%3A%2F%2Fandroid.googlesource.com%2Fplatform%2Fsuperproject%3Flang%3Dgo%3Fpath%3Dbuild%2Fsoong%2Fandroid%23var%2520%255Bpaths.go%2523609%255D.depErr), [ok](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=1;l=609?gsn=ok&gs=kythe%3A%2F%2Fandroid.googlesource.com%2Fplatform%2Fsuperproject%3Flang%3Dgo%3Fpath%3Dbuild%2Fsoong%2Fandroid%23var%2520%255Bpaths.go%2523609%255D.ok) := [err](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=603).([missingDependencyError](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=620)); [ok](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=609) {
			[missingDeps](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=599) = append([missingDeps](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=599), [depErr](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=609).[missingDeps](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=621)...)
		} else if [err](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=603) != nil {
			[reportPathError](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=139)([input](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=571).[Context](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=409), [err](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=603))
		}
		[expandedSrcFiles](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=601) = append([expandedSrcFiles](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=601), [srcFiles](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=603)...)
	}

	return [expandedSrcFiles](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=601), append([missingDeps](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=599), [missingExcludeDeps](https://cs.android.com/android/platform/superproject/+/master:build/soong/android/paths.go;drc=949eb5cc19e29dec8c656b11184fc441ba4702df;bpv=1;bpt=0;l=579)...)
}